package/MDAnalysis/topology/tpr/setting.py

```diff
@@ -41,7 +41,7 @@
 from six.moves import range
 
 #: Gromacs TPR file format versions that can be read by the TPRParser.
-SUPPORTED_VERSIONS = (58, 73, 83, 100, 103, 110, 112)
+SUPPORTED_VERSIONS = (58, 73, 83, 100, 103, 110, 112, 116)
```

Comment:
@jbarnoud how risky is it to allow versions greater than this, but issue a warning that this is currently speculative support?  Ie how much does the tpr format change version to version?

Reply:
The last two versions (2018 and 2019) did work with virtually no change. I am very uneasy about it, though, as the release note lists many features added and removed.

Reply:
Let's not allow greater versions. Much better to fail reliably than to work unreliably.